assignment-client/src/AssignmentClient.cpp

```diff
@@ -89,6 +90,13 @@ AssignmentClient::AssignmentClient(Assignment::Type requestAssignmentType, QStri
     _assignmentServerSocket.setObjectName("AssignmentServer");
     nodeList->setAssignmentServerSocket(_assignmentServerSocket);
 
+    if (disableDomainPortAutoDiscovery == true) {
+        _disableDomainPortAutoDiscovery = disableDomainPortAutoDiscovery;
+        qCDebug(assignment_client) << "Disabling domain port auto discovery by the assignment client due to parsed command line parameter.";
+    }
+
+    nodeList->setDomainPortAutoDiscovery(_disableDomainPortAutoDiscovery);
```

Comment:
This value seems to be backwards, shouldn't it be set to !_disableDomainPortAutoDiscovery?